tests/test_subdir.py

```diff
@@ -0,0 +1,30 @@
+"""
+Test if the subdirectory is correctly navigated to
+"""
+import logging
+from os.path import abspath, dirname
+
+import pytest
+from repo2docker.app import Repo2Docker
+
+repo_path = dirname(dirname(abspath(__file__)))
```

Comment:
Why there `dirname(dirname())`? Maybe add a comment and/or use `..` to signal "one level up"?

Reply:
I added a comment here - repo_path is the root directory of the repo2docker git repository.